--- a/lumina-desktop/LXcbEventFilter.cpp	2016-03-25 14:49:33.025831000 -0700
+++ b/lumina-desktop/LXcbEventFilter.cpp	2016-03-25 16:50:44.718633558 -0700
@@ -26,7 +26,7 @@
 }
 
 //This function format taken directly from the Qt5.3 documentation
-bool XCBEventFilter::nativeEventFilter(const QByteArray &eventType, void *message, long *) Q_DECL_OVERRIDE
+bool XCBEventFilter::nativeEventFilter(const QByteArray &eventType, void *message, long *) 
 {
 	if(stopping){ return false; } //don't do any parsing
 	//qDebug() << "New Event";
